src/components/_global/JazzIcon/addressToNumber.js

```diff
@@ -0,0 +1,4 @@
+function addressToNumber(address) {
```

Comment:
Why not just add this as a method in the Jazzicon component? Seems unnecessary to create a new file for it.

Reply:
We used to use [vue3-jazzicon](https://www.npmjs.com/package/vue3-jazzicon) which bundle is not ready for esmodules so I decided to copy the code and include it as our own component to avoid to much dark magic in our vite setup. `addressToNumber` was a separated file in the original npm package so I kept it to make it easier to update to a future version. 
I know it is not ideal but I it is not a critical component so I thought this solution was enough. What do you think?